src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -47,6 +51,33 @@ export class SkyPhoneFieldFixture {
     fixture.detectChanges();
   }
 
+  /**
+   * Blurs the select and returns a void promise that indicates when the action is complete.
+   */
+  public async blur(): Promise<void> {
```

Comment:
What if the underlying trigger changes from 'blur' to 'click' (as an example)? Our fixture API would be stuck on using `blur` when it doesn't necessarily correlate to the actual event. Is there a more generic name that we could use here? Why is it needed as a public method?

Reply:
My intent here wasn't to tie into the underlying trigger. It was just to offer a way to easily blur the input based on two things: 1) I'm seeing consumers write their own code to blur the input in their tests. 2) I see this "blur" pattern on a lot of test harnesses from angular material: https://github.com/angular/components/blob/master/src/material/input/testing/input-harness.ts#L101

Reply:
Cool, cool. Thanks for the link.